ruby/tests/well_known_types_test.rb

```diff
@@ -139,4 +139,31 @@ def test_any
     assert any.is(Google::Protobuf::Timestamp)
     assert_equal ts, any.unpack(Google::Protobuf::Timestamp)
   end
+
+  def test_struct_assign
+    s = Google::Protobuf::Struct.new(fields: {'a' => Google::Protobuf::Value.new({number_value: 4.4})})
+    assert_equal 4.4, s['a']
+
+    s = Google::Protobuf::Struct.new(fields: {'a' => {number_value: 2.2}})
```

Comment:
i would love to see the `fields` Hash keys be converted to Strings, so we could also use this:

```ruby
s = Google::Protobuf::Struct.new(fields: {a: {number_value: 2.2}})
```

Reply:
you got it